clap_mangen/tests/testsuite/roff.rs

```diff
@@ -96,6 +96,13 @@ fn sub_subcommands_help() {
     }
 }
 
+#[test]
+fn help_headings() {
+    let name = "my-app";
+    let cmd = common::help_headings(name);
+    common::assert_matches(snapbox::file!["../snapshots/help_headings.bash.roff"], cmd);
+}
```

Comment:
Per our contributor guide, would you mind changing the commits of your PR so that they are
1. This test, passing, showing the behavior before your change
2. The feature work with the update to the test so it will pass with the new behavior